demo/api/schema.gql

```diff
@@ -1166,6 +1166,7 @@ type Mutation {
   createProductCategory(input: ProductCategoryInput!): ProductCategory!
   updateProductCategory(id: ID!, input: ProductCategoryUpdateInput!): ProductCategory!
   deleteProductCategory(id: ID!): Boolean!
+  setProductCategory(id: ID!, input: SetProductCategoryInput!): ProductCategory!
```

Comment:
instead use the existing updateProductCategory; input is disabled for products relation in ProductCategory entity, just enable it